src/content/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent.mdx

```diff
@@ -325,6 +325,21 @@ For other message queue libraries, use [custom instrumentation](/docs/agents/nod
 * [Microsoft Azure](/docs/nodejs/nodejs-agent-on-microsoft-azure)
 * [Heroku](/docs/nodejs/nodejs-agent-on-heroku)
 
+## HTTP/1.1
+
+The Node.js agent instruments the native `http` module. In addition, the popular [undici](https://www.npmjs.com/package/undici) client is also supported.
```

Comment:
as mentioned in standup.  currently undici is not generally available.  you must set `feature_flag.undici_instrumentation = true`